src/main/ui/webpack.config.js

```diff
@@ -46,6 +46,9 @@ module.exports = function (env) {
             new ExtractTextPlugin(CSS_FILENAME),
             new HtmlWebpackPlugin({
                 template: 'index.ejs'
+            }),
+            new webpack.DefinePlugin({
+                GA_ID: env === 'prod' ? JSON.stringify('UA-16695719-3') : undefined
```

Comment:
This is not right. This is supposed to be a secret, and it should not live in the code at all, but in an external config file (which is set up for the deployment).
I would expect a line in the README file to explain that this can be set.

(Sorry for the delay I had missed the email notifications on this so I just saw it when looking at another PR.)

Reply:
@djazayeri Is these certain file which includes env variables or should I create my own?

Reply:
@reubenvarghese1 do you have any response to the above?

Reply:
@BartlomiejRasztabiga are you now in position to continue? 😊

Reply:
@BartlomiejRasztabiga You need not worry about this issue if you are busy. We just need to know if you will be working on this. If not, we will make it ready for work so that someone in the future may work on it. This helps us prevent stagnant issues :-)

Reply:
I want to complete this but first I need to know if there currently is some way of setting env variables during deployment or can I implement my system, e.g. using method @reubenvarghese1 provided in the comment above.

Reply:
I want to complete this but first I need to know if there currently is some way of setting env variables during deployment or can I implement my system, e.g. using method @reubenvarghese1 provided in the comment above.

Reply:
@BartlomiejRasztabiga Makes sense. Since I am not aware of what happens during Deployment, I would wait for @djazayeri to clear your doubt .

Reply:
Hmm, looking at this again I realize a few things:
1. we don't have a code example of this yet. (all configurable things so far are on the server-side, not the client code)
2. this id isn't _super_ secret because a determined attacker could probably get it from the JS.

So I can live with you doing it the way you've done here.